ci_setup.jl

```diff
@@ -0,0 +1,21 @@
+VERSION >= v"0.7.0-" && using InteractiveUtils
+versioninfo()
+
+if VERSION < v"0.7.0-"

```

Comment:
Because your `Project.toml` says you need at least Julia 1.3, you don't need to have these kinds of conditionals anymore.

Reply:
oh, I don't think this file is used at all anymore anyways. delete-fest.